src/main/java/smokefree/domain/Initiative.java

```diff
@@ -50,6 +57,9 @@
 
     @CommandHandler
     public Initiative(CreateInitiativeCommand cmd, MetaData metaData) {
+        validateMaximumPlaygroundCapacity();
+        validateDuplicatePlaygroundNames(cmd.getName());
+        validatePlaygroundsRange(cmd.getGeoLocation(), SmokefreeConstants.MAXIMUM_PLAYGROUNDS_DISTANCE);
```

Comment:
Note the inconsistency here. The MAXIMUM_PLAYGROUNDS_ALLOWED is accessed in the validation method, while the MAXIMUM_PLAYGROUNDS_DISTANCE is passed in as a parameter. 
Is there a reason for this?

Reply:
unable to find this.